### PR TITLE
Improvement/add endpoint returning scheduler services

### PR DESCRIPTION
**Pull request by eMKoded**

### CODE REVIEW

File: services/scheduler/src/app/features/scheduling/actions/get-services.action.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+export const getServicesActionValidation = celebrate(
+  {
+    headers: Joi.object(),
+  },
+  { abortEarly: false },
+);
+
+/**
+ * @swagger
```
Comment (mkopa) [inline services/scheduler/src/app/features/scheduling/actions/get-services.action.ts:18]: swagger definitions missing

File: services/scheduler/src/config/manifest.json [hunk 2]
```diff
@@ -4,9 +4,29 @@
     "name": "security",
```
Comment (mkopa) [inline services/scheduler/src/config/manifest.json:4]: Add manifest.json validator to container.ts just like we validate the AppConfig (or initialUsersData from security service)

File: services/scheduler/src/app/features/scheduling/actions/get-services.action.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+export interface GetServicesActionProps {
+  commandBus: CommandBus;
+}
+
+export const getServicesActionValidation = celebrate(
+  {
+    headers: Joi.object(),
+  },
+  { abortEarly: false },
+);
```
[the remark this answers is not in the document]
Comment (eMKoded) [reply services/scheduler/src/app/features/scheduling/actions/get-services.action.ts:15]: This endpoint has no params, so there's nothing to validate

File: services/scheduler/src/config/config.ts [hunk 4]
```diff
@@ -23,6 +23,24 @@ export const appConfigSchema = Joi.object({
...
+  .items(
+    Joi.object().keys({
+      url: Joi.string().required(),
+      name: Joi.string().required(),
+      actions: Joi.array().items(
+        Joi.object().keys({
+          type: Joi.string().required(),
+          http: Joi.object().keys({
+            uri: Joi.string().required(),
+            method: Joi.string().valid("GET", "POST", "PUT", "PATCH", "DELETE"),
```
Comment (mkopa) [inline services/scheduler/src/config/config.ts:36]: Should be required and I think that should be a full set of methods (I mean `GET, HEAD, POST, PUT, DELETE, CONNECT, OPTIONS, TRACE, PATCH`)

Comment (sethii) [reply services/scheduler/src/config/config.ts:36]: I doubt we need OPTIONS, TRACE or CONNECT - we are talking about scheduling things, so you either use GET/POST/PUT/DELETE 

even HEAD is not so often, but lets allow it

**Pull request merged**